superset-frontend/plugins/legacy-plugin-chart-event-flow/package.json

```diff
@@ -34,6 +34,10 @@
     "prop-types": "^15.6.2"
   },
   "peerDependencies": {
-    "react": "^15 || ^16"
+    "react": "^15 || ^16",
+    "@superset-ui/chart-controls": "0.18.25",
+    "@superset-ui/core": "0.18.25",
+    "react": "^16.13.1",
```

Comment:
The "react" duplicated

Reply:
Hmm, it only seem to build with both otherwise I ended up with major errors.